### PR TITLE
Split changelog into multiple posts

### LINKED ISSUE #6: Creating the changelog doesn't work if contributors list is too long
Reproducible with `/community changelog mattermost 2019-10`

### PR DESCRIPTION
**Pull request by hanzei**
Fixes #6

### CODE REVIEW

**Comment by amyblais** [comment]
@hanzei Do you know who will review this or can we request a reviewer?

**Comment by hanzei** [comment]
@cpoile Could you please give this a review? I know you are not familiar with the code base, so take your time and reach out to me if you have any questions.

File: server/changelog.go [hunk 1]
```diff
@@ -115,12 +115,14 @@ func (p *Plugin) updateChangelogPost(post *model.Post, userID, org, repo string,
...
+		const userPerPost = 150
+		commiterTexts := make([]string, len(commiter)/userPerPost+1)
 		for i, c := range commiter {
-			commiterText += fmt.Sprintf("[%[1]s](https://github.com/%[1]v)", c)
+			profile := fmt.Sprintf("[%[1]s](https://github.com/%[1]v)", c)
 			if i+1 != len(commiter) {
-				commiterText += ", "
+				profile += ", "
 			}
+			commiterTexts[i/150] += profile
```
Comment (cpoile) [inline server/changelog.go:125]: That's clever, nice.

**Review by cpoile — APPROVED** [review]

**Comment by cpoile** [comment]
@hanzei I'll leave this to you (not sure if you want another review before merging?)

**Comment by hanzei** [comment]
@amyblais I will deploy a new version of the plugin on ci-extensions once https://community-release.mattermost.com/core/pl/jg1qmj1ogfnwtqrxnm6n63rbgr is resolved.

**Comment by hanzei** [comment]
@amyblais The command now works fine

**Pull request merged**